sdk/cosmos/azure_data_cosmos/examples/cosmos/create.rs

```diff
@@ -4,42 +4,129 @@ use azure_data_cosmos::{
     clients::{ContainerClientMethods, DatabaseClientMethods},
     CosmosClient, CosmosClientMethods, PartitionKey,
 };
-use clap::Args;
+use clap::{Args, Subcommand};
 
-/// Creates a new item.
+#[cfg(feature = "control_plane")]
```

Comment:
Annoyingly, it sees we need to conditionally-compile our `use` statements as well to avoid 'unused import' lints. I'd prefer it if clippy considered all features (maybe we just need to run it with `--all-features` 🤔 ) since extraneous imports don't impact the functionality, but alas.